src/ema/tests/invariants.rs

```diff
@@ -136,6 +136,24 @@ proptest! {
     }
 }
 
+proptest! {
+    #[test]
+    fn iterated_volume_ema_approaches_zero(
```

Comment:
This doesn't seem like the right test to me, at least for what I have in mind.

I would think you would want to test a scenario where the oracle was last updated (say) 5 blocks ago, and test the "5-block" update, and compare it to updating after each one of those 5 blocks.

Reply:
I added a test in the pallet that I think covers what you have in mind: https://github.com/galacticcouncil/warehouse/blob/0249d4bf94cf99d5f9e3402fc000e169fe369b65/ema-oracle/src/tests/invariants.rs#L119
If you want I can come up with a scenario test in the math package here as well, though.